src/main/java/taxiservice/controllers/cars/CreateCarController.java

```diff
@@ -0,0 +1,38 @@
+package taxiservice.controllers.cars;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import taxiservice.Application;
+import taxiservice.lib.Injector;
+import taxiservice.models.Car;
+import taxiservice.models.Manufacturer;
+import taxiservice.service.CarService;
+import taxiservice.service.ManufactureService;
+
+public class CreateCarController extends HttpServlet {
+    private static final Injector injector =
+            Injector.getInstance(Application.class.getPackageName());
+    private static final CarService carService =
+            (CarService) injector.getInstance(CarService.class);
+    private static final ManufactureService manufactureService =
+            (ManufactureService) injector.getInstance(ManufactureService.class);
+
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getRequestDispatcher("/WEB-INF/cars/create.jsp").forward(req, resp);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws IOException {
+        String model = req.getParameter("model");
+        String manufactureId = req.getParameter("manufacture id");
```

Comment:
```suggestion
        String manufactureId = req.getParameter("manufacture_id");
```
This parameter's name is different from the one in jsp. So it doesn't work. Let's test code before commiting it
![image](https://user-images.githubusercontent.com/50640794/105384569-0e605a00-5c1b-11eb-8496-1bd18dcdbe08.png)

Reply:
check this everywhere